nixos/modules/services/logging/SystemdJournal2Gelf.nix

```
@@ -25,7 +25,7 @@ in
       };
 
       extraOptions = mkOption {
-        type = types.string;
+        type = types.str;
```

```suggestion
        type = types.separatedString " ";
```
Just for consistency with postgresql-backup service, which you've changed to `separatedString`